README.md

```diff
@@ -1 +1,33 @@
-# mvblogapp
\ No newline at end of file
+# Blog App
+
```

Comment:
- [ ] I am requesting this change here because the affected file is not among the changed files. The database configuration in your `config/database.yml` file is for **SQLite**, but it should be **Postgres**. Kindly change your database to Postgres

  ![uir](https://user-images.githubusercontent.com/81468675/205095083-08f98f18-feae-4112-80d0-ec509104fcba.jpg)

- [ ] Your readme file looks scanty. Kindly use the actual readme that was shared with you and includes very detailed steps on how to set up and run the project in the local environment. Kindly include all the sections in the readme: [https://github.com/microverseinc/readme-template/blob/master/README.md](https://github.com/microverseinc/readme-template/blob/master/README.md)

  Kindly note that only Deployment, tests, and live link sections are not compulsory since they are not available at this time.

  ![image](https://user-images.githubusercontent.com/81468675/205097428-18407fda-f70b-4aea-9125-9c7abacf5f0b.png)

Reply:
Thank you for your help @julie-ify 
- I have install PostgresSQL as the database
- I also updated README file. (I have consider this is not necessary to be a block, since there are some information provided.
- And also, I have not completed the project.)